generator/lib/platform/PgsqlPlatform.php

```diff
@@ -59,22 +59,6 @@ public function getMaxColumnNameLength()
         return 32;
     }
 
-    /**
-     * Escape the string for RDBMS.
-     *
-     * @param string $text
-     *
-     * @return string
-     */
-    public function disconnectedEscapeText($text)
```

Comment:
`pg_escape_string` in PHP 8 requires connection to be passed, but this method is `disconnectedEscapeText`, so there will never be a connection available. Method falls back to parent implementation of `disconnectedEscapeText`, so that' why I just removed it.